### ISSUE TITLE
Cannot enable JIT for "Play!" Emulator. Shortcut says "Could not find 'Play!'"

### ISSUE THREAD

**Issue by DelayedHappiness** [nythepegasus/SideJITServer#27]
![image](https://github.com/nythepegasus/SideJITServer/assets/91527381/83baddf6-26fb-4194-a4c6-6c07ab64718f)

![image](https://github.com/nythepegasus/SideJITServer/assets/91527381/65fe9398-57a7-44f7-904b-7ea69b65e78a)

**Comment by DelayedHappiness** [comment]
It works for ppsspp, but when trying to enable it for play it says it can't find the app

**Comment by Montivillon** [comment]
http://127.0.0.1:8080/re/

**Comment by DelayedHappiness** [comment]
Whenever I go to that page it does not load. Any particular reason for that?

**Comment by Montivillon** [comment]
well idk for some reason, it didnt work for me last time but now it works????

**Comment by Montivillon** [comment]
its the only way to refresh the apps that dont work

**Comment by DelayedHappiness** [comment]
Never works for me lol. Honestly don't know what to say, the page just acts like it doesn't exist. But using the normal "http://127.0.0.1:8080 does work

**Comment by Montivillon** [comment]
> Whenever I go to that page it does not load. Any particular reason for that?

I KNOW. your not supposed to use a mobile device in that link its only the computer

**Comment by DelayedHappiness** [comment]
> its the only way to refresh the apps that dont work
> 
> 

I guess I'm out of luck for playing ps2 then...

**Comment by Montivillon** [comment]
> Never works for me lol. Honestly don't know what to say, the page just acts like it doesn't exist. But using the normal "http://127.0.0.1:8080" does work

i also use it but not used to refresh

**Comment by DelayedHappiness** [comment]
> > Whenever I go to that page it does not load. Any particular reason for that?
> 
> 
> 
> I KNOW. your not supposed to use a mobile device in that link its only the computer
> 
> 

Yeah no, I've used it on both... neither the computer nor the phone work. Page doesn't exist to either

**Comment by Montivillon** [comment]
use http://127.0.0.1:8080/re/ and open the website in the computer

**Comment by DelayedHappiness** [comment]
> Never works for me lol. Honestly don't know what to say, the page just acts like it doesn't exist. But using the normal "http://127.0.0.1:8080" does work

I'ma try. Let you know if anything changes and I'll also send a screenshot of the page

**Comment by Montivillon** [comment]
Maybe just type it

**Comment by Montivillon** [comment]
While ur here, can you fix this
![Uploading IMG_0199.jpeg…]()

**Comment by DelayedHappiness** [comment]
Well, it shows this with the server active and inactive. 
![Uploading Screenshot 2024-04-25 at 20-13-29 Problem loading page.png…]()

Also, fix what? the image link sends me to this issue page

**Comment by DelayedHappiness** [comment]
@Montivillon I got it to refresh with the link you said, it still says the same thing sadly. maybe a restart is needed

**Comment by DelayedHappiness** [comment]
Neither a restart of the server nor the phone work either. sadly this seems to be something else entirely, but the server did seem to almost work when i restarted it tho. just got stuck and then send an internal error message after multiple tries

**Comment by DelayedHappiness** [comment]
![image](https://github.com/nythepegasus/SideJITServer/assets/91527381/f89a7444-74e9-4def-a641-b6eabffa8b03)

This is what I get now lol

**Comment by cootshk** [comment]
@DelayedHappiness Same issue, trying to use DolphiniOS

**Comment by Montivillon** [comment]
> ![image](https://private-user-images.githubusercontent.com/91527381/326000561-f89a7444-74e9-4def-a641-b6eabffa8b03.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQxNzc1MzEsIm5iZiI6MTcxNDE3NzIzMSwicGF0aCI6Ii85MTUyNzM4MS8zMjYwMDA1NjEtZjg5YTc0NDQtNzRlOS00ZGVmLWE2NDEtYjZlYWJmZmE4YjAzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI3VDAwMjAzMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWMyZjkzMzMxNDk0OWRkZjA2YzZjNjdhMjQxZjVmZTExNTA0Y2U4MGI1MmQxZGE3ZTI3MDg1MGU5ZGVhZGZkNzImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.8q7tqtPTM9Z6oS2Oea_9jF5N-On8K6c2HqGinIw29Q0)
> 
> This is what I get now lol

That’s my problem even when refreshing which is what I asked you to fix

**Comment by DelayedHappiness** [comment]
@Montivillon yeah I see what you mean... it's just a dinosaur saying there's no response from the server anyway. But if you mean what I get on the terminal then shit, idk either that's why I'm here lol

**Comment by FlMihai** [comment]
I´m using SideStore 0.5.5 nightly with SideJITSupport and it works, otherwise I´m getting the same error if I try to activate JIT from the shortcut.

**Comment by DelayedHappiness** [comment]
@FlMihai And which discord would that be? If you don't mind me asking

**Comment by jhonmirror** [comment]
> you´ll just need to add your iCloud termination/signature at the end of the app package name.

Could you kindly explain in practice how I should do it?

**Comment by nythepegasus** [comment]
I think @stossy11 has fixed this, but if not definitely reopen.

**Comment by herolsl** [comment]
> I think @stossy11 has fixed this, but if not definitely reopen.

I think it's a shortcut problem. The name of Play! contains !, and the shortcut only matches Play, so it finally prompts Could not find 'Play'. However, other apps that do not contain special symbols are normal. Maybe you can fix it.

**Comment by herolsl** [comment]
> I think @stossy11 has fixed this, but if not definitely reopen.

Maybe the name matches problem. Play! contains ！，other ordinary name works fine.

**Comment by phcreery** [comment]
@FlMihai 

FYI, discord is not a wiki.

@DelayedHappiness @jhonmirror 

[SideJIT-Play-Test1.shortcut.zip](https://github.com/user-attachments/files/22705246/SideJIT-Play-Test1.shortcut.zip)

I have spent the last 20 minutes digging through discord for the "Fixed Shortcut".
I found it, and attached it here as it should have been a year ago, but it still needs further modification.
1. Install this shortcut as normal.
2. Goto: http://127.0.0.1:8080/
3. Copy your ID (`00008101-xxxxxxxxxxxx...`)
4. Goto: `http://127.0.0.1:8080/00008101-xxxxxxxxxxxx...`
5. Copy the play bundle value (`com.virtualapplications.play.3532B38PB3`)
6. Modify shorcut and find the section "Get contents of server / UDID". Edit it to match the bundle value from previous step
7. Run shortcut
